include/rnp/rnp.h

```diff
@@ -79,6 +79,17 @@ typedef uint32_t rnp_result_t;
 #define RNP_LOAD_SAVE_PERMISSIVE (1U << 8)
 #define RNP_LOAD_SAVE_SINGLE (1U << 9)
 
+/**
+ * Flags for the rnp_key_remove_signatures
+ */
+
+#define RNP_KEY_SIGNATURE_INVALID (1U << 0)
+#define RNP_KEY_SIGNATURE_UNKNOWN_KEY (1U << 1)
+#define RNP_KEY_SIGNATURE_NON_SELF (1U << 2)
```

Comment:
I know these are *somewhat* self-explanatory, but I still think there's slight ambiguity so can we document these more?

Reply:
@dewyatt Defines are documented in the description of `rnp_key_remove_signatures()`. Anyway, let's rename it to the `RNP_KEY_SIGNATURE_NON_SELF_SIG`, looks better.